### PR TITLE
Sp2 user view rating

### PR DESCRIPTION
**Pull request by Spellbreeze**
Pair Programming: Parkaranjot Singh, Madhura Gangal, Alexander Goebel

### CODE REVIEW

File: backend/controllers/ratings_controller.js [hunk 1]
```diff
@@ -53,33 +53,84 @@ export const getRating = asyncHandler(async(req, res) => {
...
       ]
       const reviews = await Ratings.aggregate(pipeline)
       res.status(200).json(reviews)
   } catch (error) {
       res.status(400).send(error)
   }
+})
+
+
+// doesnt work code
```
Comment (madhura112) [inline backend/controllers/ratings_controller.js:121]: may want to remove unnecessary code

File: backend/models/ratings_model.js [hunk 2]
```diff
@@ -20,8 +20,27 @@ const ratingsSchema = new Schema({
...
+        type: String,
+        required: true
+    },
+    user_name: {
+        type: String,
+        required: true
     }
 });
+// not needed
+ratingsSchema.statics.findByEmail = async (email) => {
```
Comment (madhura112) [inline backend/models/ratings_model.js:34]: Can get rid of unused code

File: backend/controllers/ratings_controller.js [hunk 3]
```diff
@@ -53,33 +53,84 @@ export const getRating = asyncHandler(async(req, res) => {
...
       const reviews = await Ratings.aggregate(pipeline)
       res.status(200).json(reviews)
   } catch (error) {
       res.status(400).send(error)
   }
+})
+
+
+// doesnt work code 
+export const getUserReviews2 = asyncHandler(async(req, res) => {
```
Comment (parkaransingh) [inline backend/controllers/ratings_controller.js:122]: Remove this in future commits please! Study why it didnt work.

**Review by madhura112 — APPROVED** [review]
Code looks good!

File: backend/server.js [hunk 4]
```diff
@@ -16,8 +16,8 @@ import userRouter from './routes/users_route.js';
 
 app.use(cors());
 
-mongoose.connect('mongodb://127.0.0.1:27017/CivilityOptics', { useNewUrlParser: true });
-
+//mongoose.connect('mongodb://127.0.0.1:27017/CivilityOptics', { useNewUrlParser: true });
+mongoose.connect('mongodb+srv://CivilityOptics:Dontpush2134@civility1.7zzflo5.mongodb.net/?retryWrites=true&w=majority', { useNewUrlParser: true });
```
Comment (parkaransingh) [inline backend/server.js:20]: Remove this password secret.

**Review by parkaransingh — APPROVED** [review]
You can merge!

**Pull request merged**